modules/nlu/src/backend/training-pipeline.ts

```diff
@@ -223,12 +224,15 @@ const TrainIntentClassifier = async (
     )
 
     const nAvgUtts = Math.ceil(_.meanBy(trainableIntents, 'utterances.length'))
+
+    const lo = getSeededLodash(process.env.RANDOM_SEED)
```

Comment:
can you create an instance of lo only once like the preprocess step ?

maybe declare global var in top of file with `let lo` then instantiate it in preprocess step and reset seed in a cleanup step ?

Reply:
Yes, that's what I first tried and it worked on Windows but did not on Ubuntu... I can make few more tests and see what I can do to make this cleaner...

Reply:
I meant at the begining of training pipeline and at the end of predict pipeline... 

you could change the tools object and always use the seeded lodash

Reply:
:+1: